packages/expo-task-manager/android/src/main/java/expo/modules/taskManager/TaskManagerPackage.java

```diff
@@ -10,19 +10,27 @@
 import org.unimodules.core.interfaces.InternalModule;
 import org.unimodules.core.interfaces.SingletonModule;
 
+import expo.loaders.provider.AppLoaderProvider;
+import expo.modules.taskManager.apploader.RNHeadlessAppStarter;
+
 public class TaskManagerPackage extends BasePackage {
+
+  static {
+    AppLoaderProvider.registerLoader("react-native-experience", RNHeadlessAppStarter.class);
```

Comment:
Again, it's a bit confusing that we register a loader (`registerLoader`) but pass in the class that is called as a starter 😅